crates/sui-benchmark/src/drivers/bench_driver.rs

```diff
@@ -373,6 +381,7 @@ impl Driver<(BenchmarkStats, StressStats)> for BenchDriver {
                 duration: Duration::ZERO,
                 num_error_txes: 0,
                 num_success_txes: 0,
+                num_expected_error_txes: 0,
```

Comment:
I'm open to doing both.. it would be nice though to have the additional metric so we can differentiate. But wasn't sure how `num_success_txes` as a data point is used. What do you think?